src/main/java/salesforce/ui/pages/classic/contracts/ClassicNewContractPage.java

```diff
@@ -110,104 +120,132 @@ public void setContractStartDate(final String newContractStartDate) {
     public void setContractTermMonths(final String newContractTermMonths) {
         this.contractTermMonths.sendKeys(newContractTermMonths);
     }
+
     /**
      * Sets the ownerExpirationNotice element.
      *
      * @param newOwnerExpirationNotice to be typed.
      */
-    public void setOwnerExpirationNotice(final WebElement newOwnerExpirationNotice) {
-        this.ownerExpirationNotice = newOwnerExpirationNotice;
+    public void setOwnerExpirationNotice(final String newOwnerExpirationNotice) {
```

Comment:
It can be a string with format to replace value that is required.

Reply:
This class is for Classic Version, for now im working on the Ligthning version and the refactor will focos on this, after the refactor is done this classes will use the WebelementAction instead of this formats